flink-connectors/flink-connector-hive/pom.xml

```diff
@@ -549,6 +549,13 @@ under the License.
 			<scope>test</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-connector-base</artifactId>
```

Comment:
What does the hive connector need it for? because of some format?

Reply:
I assume so. Without this dependency the test `HiveInputFormatPartitionReaderITCase.testReadMultipleSplits()` fails with:
```
Caused by: java.lang.NoClassDefFoundError: org/apache/flink/connector/base/source/reader/SingleThreadMultiplexSourceReaderBase
```